composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": ">=5.5",
         "container-interop/container-interop": "^1.1",
-        "zendframework/zend-expressive": "^1.0@rc",
+        "zendframework/zend-expressive": "~1.0.0-dev@dev",
```

Comment:
my assumption is this may be problematic when zend-expressive need to download zendviewrenderer. But I am not exactly sure if this will work without minimum-stability flag in expressive. How about you add minimum-stability dev to the composer.json and keep the rest in tact ?